File: w09/d03/jaclyn_jimenez/mail_app/app/assets/javascripts/package.js
function Package() {
  this.content = [];
  this.fragile = true;
  this.broken = false;
}

Package.prototype.shake = function() {
  if (this.fragile) {
    this.broken = true;
  }
};
Package.prototype.isBroken = function() {
  return this.broken;
};

File: w09/d01/alex_hint/mail_items/lib/package.rb
class Package < MailItem

  attr_accessor :content, :fragile, :broken

  def initialize(options = {})
    @content = options[:content]
    @fragile = false
    @broken = true
  end

  def is_fragile?
    @fragile
  end

  def shake
    if @fragile
      @broken = true
    end

  end

  def is_broken?
    @broken
  end
end